02/Form.js

```diff
@@ -1,13 +1,19 @@
-import React from 'react';
+import React from "react";
 
 class Form extends React.Component {
-    render() {
-        return (
-            <section>
-                <input /><input type="submit" />
-            </section>
-        )
-    }
+  handleSubmitForm(e) {
+    e.preventDefault();
+    this.props.onAddUser(this.input.value);
+  }
+
+  render() {
+    return (
+      <form onSubmit={(e) => this.handleSubmitForm(e)}>
+        <input ref={(node) => (this.input = node)} />
```

Comment:
Raczej nie piszemy nawiasu jak jest tylko 1 parametr.
Nawiasy wew. funkcji też są zbędne ;)

Reply:
Wiem, że nie trzeba, to mi domyślnie dodaje Prettier.